File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/RecordsVo.java
package org.javamaster.invocationlab.admin.model.erd;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

/**
 * @author yudong
 * @date 2023/2/14
 */
@NoArgsConstructor
@Data
public class RecordsVo {
    private String id;
    private String projectName;
    private String tags;
    private String description;
    private Integer type;
    private String creator;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private Date createTime;
    private String updater;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private Date updateTime;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/RecordsVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import com.fasterxml.jackson.annotation.JsonFormat
import java.util.*

/**
 * @author yudong
 * @date 2023/2/14
 */


@AllOpen
class RecordsVo {
    var id: String? = null
    var projectName: String? = null
    var tags: String? = null
    var description: String? = null
    var type: Int? = null
    var creator: String? = null

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    var createTime: Date? = null
    var updater: String? = null

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    var updateTime: Date? = null
}
